src/components/ToggleSwitch/index.tsx

```diff
@@ -33,22 +34,30 @@ const styles = StyleSheet.create({
   textOn: {
     fontFamily: 'Inter',
     fontWeight: '400',
-    fontSize: dimen.normalizeDimen(12),
+    fontSize: normalizeFontSize(12),
     lineHeight: dimen.normalizeDimen(18),
     textAlign: 'center',
+    textAlignVertical: 'center',
     color: '#f4f3f4',
     position: 'absolute',
-    left: dimen.normalizeDimen(2)
+    top: dimen.normalizeDimen(1),
+    left: dimen.normalizeDimen(2),
+    width: dimen.normalizeDimen(22),
+    height: dimen.normalizeDimen(18)
   },
   textOff: {
     fontFamily: 'Inter',
     fontWeight: '400',
-    fontSize: dimen.normalizeDimen(12),
+    fontSize: normalizeFontSize(12),
     lineHeight: dimen.normalizeDimen(18),
     textAlign: 'center',
+    textAlignVertical: 'center',
     color: '#f4f3f4',
     position: 'absolute',
-    right: dimen.normalizeDimen(2)
+    top: dimen.normalizeDimen(1),
+    right: dimen.normalizeDimen(2),
+    width: dimen.normalizeDimen(22),
+    height: dimen.normalizeDimen(18)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `textOn` and `textOff` styles are significant. The introduction of `textAlignVertical`, `top`, `left`, `right`, `width`, and `height` properties will help in better positioning and alignment of the text within the toggle switch. However, it's important to ensure that these changes don't break the layout on different screen sizes or orientations.
```diff
+ textAlignVertical: 'center',
+ top: dimen.normalizeDimen(1),
+ left: dimen.normalizeDimen(2),
+ width: dimen.normalizeDimen(22),
+ height: dimen.normalizeDimen(18)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->